packages/iov-core/src/workers/signingservice.worker.ts

```diff
@@ -16,6 +16,10 @@ const bnsdFaucetMnemonic = "degree tackle suggest window test behind mesh extra
 const ethereumUrl = "http://localhost:8545";
 const ganacheMnemonic = "oxygen fall sure lava energy veteran enroll frown question detail include maximum";
 
+async function sleep(ms: number): Promise<void> {
```

Comment:
Isn't this function in some util package now? I've seen it a dozen places

Reply:
Yes it exists in a dozen places across multiple packages and there is no package for that